### PR TITLE
Update community-plugins.json

### PR DESCRIPTION
**Pull request by Quorafind**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/Quorafind/Obsidian-Bionic-Reading

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
You can remove `obsidian` from the plugin id, its unecessary.

Please use a different name for the plugin, "Bionic Reading" is already copyrighted/trademarked.
I would recommend removing every reference of that name in code.

You mention in the readme that the plugin works in reading mode, where is this implemented?

**Comment by Quorafind** [comment]
@joethei I think `A plugin to enable bionic reading mode in Live preview mode of Obsidian.` is equal to a plugin that enable `Bionic reading mode` in `live preview mode` of Obsidian.

I will rename bionic reading to another name, but I don't think I have to remove every reference of that name (or maybe just in code?). Someone wants to search bionic reading in Obsidian so that I have to leave at least one in README or plugin's description.

**Comment by Quorafind** [comment]
Finished @joethei

**Comment by liamcain** [comment]
- [linePos](https://github.com/Quorafind/Obsidian-Better-Reading-Mode/blob/61922663e82c10f0cf49a51831750202c2a8b922/src/betterReadingWidget.ts#L81) I don't think you need the `?` optional unwrap here. lineAt doesn't return null

File: community-plugins.json [hunk 1]
```diff
@@ -5409,5 +5409,12 @@
         "description": "A plugin to enable using obsidian notes as calculation sheets.",
         "author": "monesga",
         "repo": "monesga/obsidian-sigma"
+    },
+    {
+        "id": "obsidian-bionic-reading-plugin",
```
Comment (liamcain) [inline community-plugins.json:5414]:
```suggestion
        "id": "better-reading-mode",
```

**Pull request merged**